.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - nightly
```

Comment:
Any reason for not building on all channels?

Reply:
No good ones! I'll add that back.